data-structures/dictionaries_new/python/dictionary_access.py

```diff
@@ -0,0 +1,20 @@
+# Raspberry Pi Foundation
+# Developed as part of Ada Computer Science
+# 
+# Usage licensed under CC BY-NC-SA 4.0
+    
+
+def main():
+    # Declare and initialise the astronaut dictionary
+    astronaut = {"craft": "ISS", "name": "Sultan Al Neyadi", "days": "14"}
```

Comment:
do we wantto stick with the day value being a string? It might be better to have an integer so they don't just think all values have to be strings?

Reply:
In C# / VB / Java you can't store mixed data types as either the key or the value!

Reply:
The alternative would be to use the astronaut "id" as an integer for the key